snapshots/devmapper/snapshotter_test.go

```diff
@@ -165,3 +140,85 @@ func TestMkfsXfsNonDefault(t *testing.T) {
 	err := mkfs(ctx, "xfs", "noquota", "")
 	assert.ErrorContains(t, err, `mkfs.xfs couldn't initialize ""`)
 }
+
+func TestMultipleXfsMounts(t *testing.T) {
+	testutil.RequiresRoot(t)
+
+	logrus.SetLevel(logrus.DebugLevel)
+
+	ctx := context.Background()
+	ctx = namespaces.WithNamespace(ctx, "testsuite")
+	tempDir, err := os.MkdirTemp("", "snapshot-suite-usage")
```

Comment:
You can use t.TempDir instead

Reply:
I believe, we simply didn't have t.TempDir when we first wrote that code. It is added in Go 1.15.

Reply:
Probably. However i still believe the original logic works better. For instance we can easily locate the path when we have to debug a failing test case with an explicitly specified dir pattern. I actually found it useful when I wrote this unit test. But if you have a strong opinion about this I am fine with making the change.